frontend/src/components/toast/toast-pie-chart.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import Style from "style-it";
+
+import { PieChart, Pie } from "recharts";
+
+import { numWithCommas } from "../../utils/plan-utils";
+
+class ToastPieChart extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.getLabel = this.getLabel.bind(this);
+  }
+
+  getLabel(e) {
```

Comment:
is it possible to remove the line coming from the label? (and just show the number next to the section of the chart)